docker/Dockerfile

```diff
@@ -7,9 +7,6 @@ RUN mkdir -p /opt && \
     curl -L -J https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.gz | tar -xz && \
     cd boost_1_67_0 && \
     ./bootstrap.sh --prefix=./ &&\
-    CC=/usr/bin/gcc CXX=/usr/bin/g++ &&\
-    echo "using gcc : fdb : ${CXX} ;" >> ./tools/build/src/user-config.jam &&\
-    cat ./tools/build/src/user-config.jam &&\
-    ./b2 toolset=gcc-fdb install --with-filesystem --with-system
+    ./b2 install --with-filesystem --with-system
```

Comment:
You should increment the LABEL version=0.0.1 line to 0.0.2

Reply:
You should update the version number of the Dockerfile within the docker/docker-compose.yaml file to the new version included within the docker/Dockerfile

Reply:
dc7996e

Reply:
LGTM